README.md

```diff
@@ -184,8 +186,8 @@ For advanced usage please take a look into our [example project](https://github.
 
 | Method                                     |                                                                                                         Description                                                                                                          | Platform |
 | ------------------------------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------: |
-| `setPage(index: number)`                   |                                                                      Function to scroll to a specific page in the PagerView. Invalid index is ignored.                                                                       |   both   |
-| `setPageWithoutAnimation(index: number)`   |                                                                      Function to scroll to a specific page in the PagerView. Invalid index is ignored.                                                                       |   both   |
+| `setPage(index: number)`                   |                                                                      Function to scroll to a specific page in the PagerView. Invalid index is ignored. The recommended way is using the `page` and `animated` props                                                         |   both   |
```

Comment:
I think we might need a broader explanation of the controlled vs uncontrolled usage in docs. But this can be added later on before merging 7.0.0 to master

Reply:
Yep, I agree! Version `7.0.0` needs some changes in docs, so it can be done later